src/openrct2/paint/track/coaster/LatticeTriangleTrack.cpp

```diff
@@ -18421,6 +18421,21 @@ static void LatticeTriangleTrackDiagBrakes(
     PaintUtilSetGeneralSupportHeight(session, height + kDefaultGeneralSupportHeight);
 }
 
+static void LatticeTriangleTrackDiagBooster(
+    PaintSession& session, const Ride& ride, uint8_t trackSequence, uint8_t direction, int32_t height,
+    const TrackElement& trackElement, SupportType supportType)
+{
+    constexpr ImageIndex images[2][kNumOrthogonalDirections] = {
+        { SPR_G2_LATTICE_TRIANGLE_TRACK_DIAG_BOOSTER_1, SPR_G2_LATTICE_TRIANGLE_TRACK_DIAG_BOOSTER_2,
+          SPR_G2_LATTICE_TRIANGLE_TRACK_DIAG_BOOSTER_1, SPR_G2_LATTICE_TRIANGLE_TRACK_DIAG_BOOSTER_2 },
+        { SPR_G2_LATTICE_TRIANGLE_TRACK_DIAG_BOOSTER_1, SPR_G2_LATTICE_TRIANGLE_TRACK_DIAG_BOOSTER_2,
+          SPR_G2_LATTICE_TRIANGLE_TRACK_DIAG_BOOSTER_1, SPR_G2_LATTICE_TRIANGLE_TRACK_DIAG_BOOSTER_2 },
```

Comment:
This needs to be simplified. Would like some help with it. My head's not wrapping around it.

Reply:
This is copied from the flat track w/ chain, so you're getting the chain code as well.  It should be something like this (apply proper formatting)

```
     constexpr ImageIndex images[2][kNumOrthogonalDirections] = {
        SPR_G2_LATTICE_TRIANGLE_TRACK_DIAG_BOOSTER_1, SPR_G2_LATTICE_TRIANGLE_TRACK_DIAG_BOOSTER_2,
        SPR_G2_LATTICE_TRIANGLE_TRACK_DIAG_BOOSTER_1, SPR_G2_LATTICE_TRIANGLE_TRACK_DIAG_BOOSTER_2 
    };

    TrackPaintUtilDiagTilesPaintExtra(
        session, 3, height, direction, trackSequence, images, supportType.metal);
}

```

Reply:
Made sense of it. 
`images[2][kNumOrthogonalDirections]` means `images[2][4]`, so two arrays of four items.
Removing `[2]` should be be enough in this case.